static/js/stat_var_hierarchy/stat_var_section_input.tsx

```diff
@@ -34,6 +34,30 @@ import { USA_CHILD_PLACE_TYPES } from "../tools/map/util";
 
 const TOOLTIP_TOP_OFFSET = 10;
 const TOOLTIP_RIGHT_MARGIN = 20;
+const PLACE_TYPE_MAPPING = {
+  EurostatNUTS1: "State",
+  EurostatNUTS2: "State",
+  Town: "City",
+  Village: "City",
+  Borough: "City",
+};
+const IGNORED_PLACE_TYPES = new Set([
```

Comment:
would it be easier to add an 'allowed place type' list for now? especially since we are about to scale up our imports..

Reply:
done